addons/web/static/src/views/calendar/calendar_common/calendar_common_renderer.js

```diff
@@ -68,7 +69,10 @@ export class CalendarCommonRenderer extends Component {
     };
 
     setup() {
-        this.fc = useFullCalendar("fullCalendar", this.props.isDisabled ? this.disabledOptions : this.interactiveOptions);
+        this.fc = useFullCalendar(
+            "fullCalendar",
+            this.props.isDisabled ? this.disabledOptions : this.interactiveOptions
+        );
```

Comment:
So as usual: if we do not modify the line, we do not apply linting changes (we try to keep minimal diff)
We sometimes do, but in dedicated tasks (so not here :p)
To apply in other places too :)